tests/test_proxy_use.py

```diff
@@ -202,8 +205,8 @@ def test_baseline_no_proxy(self):
     """
 
     logger.info('Trying HTTP download with no proxy: ' + self.url)
-    download.safe_download(self.url, self.target_data_length)
-    download.unsafe_download(self.url, self.target_data_length)
+    download.safe_download(self.url, self.target_data_length, self.fetcher)
```

Comment:
do the `safe_download()` and `unsafe_download()` calls in test_proxy_use.py need to close the downloaded file?

Reply:
Yes I think they should. For some reason there are no warnings about unclosed files on these though...

Reply:
Fixed in 8df22a9.